app/views/commontator/comments/_body.html.erb

```diff
@@ -3,4 +3,6 @@
   comment
 %>
 
-<%= commontator_simple_format comment.body %>
+<div id="comment-body-<%= comment.id %>">
+  <%= auto_link_comments(comment) %>
```

Comment:
```suggestion
  <%= auto_link(comment) %>
```
Why not use the inbuilt method from the `rails_autolink` gem? Seems better than writing a custom helper function `auto_link_comments`.

https://github.com/tenderlove/rails_autolink#basic-usage

Reply:
Thanks a lot for correcting my approach, I looked into the method and it is much better than helper method. I have made the changes in my new commit.